realm/realm-jni/src/io_realm_internal_TableQuery.cpp

```diff
@@ -1066,7 +1066,7 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_TableQuery_nativeFindWithHandover
 {
     TR_ENTER()
     try {
-        std::unique_ptr<Query> query = getHandoverQuery(bgSharedGroupPtr, replicationPtr, queryPtr);
+        std::unique_ptr<Query> query = getHandoverQuery(bgSharedGroupPtr, replicationPtr, queryPtr, false);
```

Comment:
I'm just curious to know if this indicates that it is safe to return an older version of object with `RealmQuery.findFirstAsync()`.